common/src/main/java/com/redhat/cloud/notifications/models/validation/AllowedHttpsScheme.java

```diff
@@ -0,0 +1,22 @@
+package com.redhat.cloud.notifications.models.validation;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Defines the annotation to make sure that the provided {@link String} field is defined with an allowed HTTP scheme.
+ */
+@Constraint(validatedBy = AllowedHttpsSchemeValidator.class)
+@Documented
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.FIELD})
+public @interface AllowedHttpsScheme {
```

Comment:
Nitpick:
```suggestion
public @interface AllowedHttpScheme {
```
Same for the validator.

Reply:
It could even be named:
```suggestion
public @interface AllowedScheme {
```

Reply:
Renamed!